_posts/pl/2021-01-28-migracja-schematow-bazy-danych.md

```diff
@@ -82,19 +86,19 @@ Aplikowania zmian wykonanych w ramach ewolucyjnej bazy danych jest już zależne
 Jeśli baza danych jest ściśle związana jedną z aplikacją, możemy ją uruchamiać bezpośrednio z kodu [🔗⁵](https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-7-running-database-migrations/#running-migrations-on-application-startup) .
```

Comment:
Jeśli baza danych jest ściśle związana **z** jedną z aplikacją, **to** możemy

Reply:
done